Template/Tag/MatomoTag.web.js

```diff
@@ -118,43 +143,230 @@
+                                break;
+                            case 'enableHeartBeatTimer':
+                                enableHeartBeatTimerIndex = k;
+                                break;
+                            case 'trackAllContentImpressions':
+                                trackAllContentImpressionsIndex = k;
+                                break;
+                            case 'trackVisibleContentImpressions':
+                                trackVisibleContentImpressionsIndex = k;
+                                break;
+                            case 'FormAnalytics::disable':
+                            case 'FormAnalytics::disableFormAnalytics':
+                            case 'FormAnalytics.disableFormAnalytics':
+                            case 'disableFormAnalytics':
+                                disableFormAnalyticsIndex = k;
+                                break;
+                            case 'MediaAnalytics::disable':
+                            case 'MediaAnalytics::disableMediaAnalytics':
+                            case 'MediaAnalytics.disableMediaAnalytics':
+                            case 'disableMediaAnalytics':
+                                disableMediaAnalyticsIndex = k;
+                                break;
+                            case 'setCustomDimension':
+                                // There could be multiple, so let's push the indexes onto an array
+                                setCustomDimensionIndexes.push(k);
+                                break;
+                        }
+                    }
+
+                    if (matomoConfig.requireCookieConsent || requireCookieConsentIndex !== -1) {
+                        tracker.requireCookieConsent();
+                        markIndexAsConfig(requireCookieConsentIndex);
                     }
 
-                    if (matomoConfig.disableBrowserFeatureDetection && typeof tracker.disableBrowserFeatureDetection === 'function') {
+                    if ((matomoConfig.disableBrowserFeatureDetection || disableBrowserFeatureDetectionIndex !== -1) && typeof tracker.disableBrowserFeatureDetection === 'function') {
                         tracker.disableBrowserFeatureDetection();
+                        markIndexAsConfig(disableBrowserFeatureDetectionIndex);
                     }
 
-                    if (matomoConfig.disableCookies) {
+                    if (matomoConfig.disableCookies || disableCookiesIndex !== -1) {
                         tracker.disableCookies();
+                        markIndexAsConfig(disableCookiesIndex);
                     }
 
-                    if (matomoConfig.enableCrossDomainLinking) {
+                    if (matomoConfig.enableCrossDomainLinking || enableCrossDomainLinkingIndex !== -1) {
                         tracker.enableCrossDomainLinking();
-                        tracker.setCrossDomainLinkingTimeout(matomoConfig.crossDomainLinkingTimeout);
```

Comment:
@snake14 Why did we removed this line ?

Reply:
Good catch. I must have accidentally left it out while reworking all of the changes. I just pushed a change adding it back in.